server/scripts/20220722000000-backfill-subscriptions.ts

```diff
@@ -0,0 +1,65 @@
+import "./bootstrap";
+import { Subscription, Document, Event } from "@server/models";
+
+const limit = 100;
+let page = parseInt(process.argv[2], 10);
+page = Number.isNaN(page) ? 0 : page;
+
+export default async function main(exit = false) {
```

Comment:
I wasn't sure if there was a recommended way to go about back-filling subscriptions, but that's when I saw https://github.com/outline/outline/blob/4f537c757829782f68c58e5c4abe384b84908ace/server/scripts/20210716000000-backfill-revisions.ts#L8
This module tries to replicate :point_up: as closely as possible.